tests/Silex/Tests/Application/TwigTraitTest.php

```diff
@@ -28,7 +28,7 @@ public function testRender()
     {
         $app = $this->createApplication();
 
-        $app['twig'] = $mailer = $this->getMockBuilder('Twig_Environment')->disableOriginalConstructor()->getMock();
+        $app['twig'] = $mailer = $this->getMockBuilder('Environment')->disableOriginalConstructor()->getMock();
```

Comment:
Shouldn't this be fully-qualified class name?

Reply:
It is. But we could use ::class probably even.